constants/chatbotCommands.py

```diff
@@ -274,6 +274,7 @@ async def removebn(fro: str, chan: str, message: list[str]) -> str:
     )
     await user_utils.set_privileges(targetID, new_privileges)
     await user_utils.remove_user_badge(targetID, badges.BEATMAP_NOMINATION)
+    await user_utils.add_user_badge(targetID, badges.ALUMNI)
```

Comment:
should only happen in the case where the user no longer has _any_ staff permissions (e.g. they may _also_ be a chat mod or admin, and just have their bn removed)

Reply:
just realized lines 272 & 273 should also act like that lol, think u could make those changes?

Reply:
!addbn should then do the opposite, removing alumni if they already have it & are becoming a staff member again